flexmeasures/api/v3_0/tests/utils.py

```diff
@@ -40,21 +40,37 @@ def get_asset_post_data(account_id: int = 1, asset_type_id: int = 1) -> dict:
     return post_data
 
 
-def message_for_post_udi_event(
+def message_for_trigger_schedule(
     unknown_prices: bool = False,
-    targets: bool = False,
+    with_targets: bool = False,
+    realistic_targets: bool = True,
+    deprecated_format_pre012: bool = False,
 ) -> dict:
     message = {
         "start": "2015-01-01T00:00:00+01:00",
-        "soc-at-start": 12.1,
-        "soc-unit": "kWh",
+        "prior": "2015-01-04T00:00:00+01:00",
```

Comment:
This now says "trigger a schedule starting January 1st, knowing what we knew prior to January 4th", and that warrants some explanation. Why did you add this particular "prior"?

Reply:
I don't recall but it isn't needed for the tests. Thanks for catching.